iceoryx_posh/source/roudi/application/roudi_app.cpp

```diff
@@ -154,10 +165,9 @@ void RouDiApp::parseCmdLineArguments(int argc, char* argv[], CmdLineArgumentPars
             std::cout << "-l, --log-level <LEVEL>           Set log level." << std::endl;
             std::cout << "                                  <LEVEL> {off, fatal, error, warning, info, debug, verbose}"
                       << std::endl;
-            std::cout << "-b,                               Start address hint of the shared memory segments"
-                         "                                  e.g. 0x1f4000000"
+            std::cout << "-c, --config-file                 Path to the RouDi Config File."
+                         "                                  Have a look at the documentation for the format."
```

Comment:
Maybe it would be nice to point to a website here instead of the generic term documentation :)

Reply:
I like this idea. Maybe we should have a FAQ where the most common issues are listed and link it here?!

Reply:
Actually from my point of view it would be fine to add a link to the markdown file in this repo, which describes how the config file works.